libs/components/src/lib/video-player/locale.ts

```diff
@@ -0,0 +1,97 @@
+	'Fullscreen'?: string;
+	'Exit Fullscreen'?: string;
+	'Picture-in-Picture'?: string;
+	'Exit Picture-in-Picture'?: string;
+	'Mute'?: string;
+	'Unmute'?: string;
+	'Playback Rate'?: string;
+	'Subtitles'?: string;
+	'subtitles off'?: string;
+	'Captions'?: string;
+	'captions off'?: string;
+	'Chapters'?: string;
+	'Close Modal Dialog'?: string;
+	'Descriptions'?: string;
+	'descriptions off'?: string;
+	'Audio Track'?: string;
+	'You aborted the media playback'?: string;
+	'A network error caused the media download to fail part-way.'?: string;
+	'The media could not be loaded, either because the server or network failed or because the format is not supported.'?: string;
+	'The media playback was aborted due to a corruption problem or because the media used features your browser did not support.'?: string;
+	'No compatible source was found for this media.'?: string ,
+	'The media is encrypted and we do not have the keys to decrypt it.'?: string;
+	'Play Video'?: string;
+	'Close'?: string;
+	'Modal Window'?: string;
+	'This is a modal window'?: string;
+	'This modal can be closed by pressing the Escape key or activating the close button.'?: string;
+	', opens captions settings dialog'?: string;
+	', opens subtitles settings dialog'?: string;
+	', opens descriptions settings dialog'?: string;
+	', selected'?: string;
+	'captions settings'?: string;
+	'Audio Player'?: string;
+	'Video Player'?: string;
+	'Replay'?: string;
+	'Progress Bar'?: string;
+	'Volume Level'?: string;
+	'subtitles settings'?: string;
+	'descriptions settings'?: string;
+	'Text'?: string;
+	'White'?: string;
+	'Black'?: string;
+	'Red'?: string;
+	'Green'?: string;
+	'Blue'?: string;
+	'Yellow'?: string;
+	'Magenta'?: string;
+	'Cyan'?: string;
+	'Background'?: string;
+	'Window'?: string;
+	'Transparent'?: string;
+	'Semi-Transparent'?: string;
+	'Opaque'?: string;
+	'Font Size'?: string;
+	'Text Edge Style'?: string;
+	'None'?: string;
+	'Raised'?: string;
+	'Depressed'?: string;
+	'Uniform'?: string;
+	'Drop shadow'?: string;
+	'Font Family'?: string;
+	'Proportional Sans-Serif'?: string;
+	'Monospace Sans-Serif'?: string;
+	'Proportional Serif'?: string;
+	'Monospace Serif'?: string;
+	'Casual'?: string;
+	'Script'?: string;
+	'Small Caps'?: string;
+	'Reset'?: string;
+	'restore all settings to the default values'?: string;
+	'Done'?: string;
+	'Caption Settings Dialog'?: string;
+	'Beginning of dialog window. Escape will cancel and close the window.'?: string;
+	'End of dialog window.'?: string;
+	'Seek to live, currently behind live'?: string;
+	'Seek to live, currently playing live'?: string;
+	'progress bar timing: currentTime={1} duration={2}'?: string;
+	'{1} is loading.'?: string;
+	'No content'?: string ;
+	'Color'?: string;
+	'Opacity'?: string;
+	'Text Background'?: string;
+	'Caption Area Background'?: string;
+	'Skip forward {1} seconds'?: string;
+	'Skip backward {1} seconds'?: string;
+	'Playing in Picture-in-Picture'?: string;
+}
```

Comment:
We don't care about these so I think it should just be opaque:
```suggestion
export type VideoPlayerLocale = unknown;
```